domains/fatgrizzly.json

```diff
@@ -0,0 +1,14 @@
+{
+  "description": "personal website ",
+  "repo": "https://github.com/fatgrizzly",
+  "owner": {
+    "username": "fatgrizzly",
+    "email": "soundar@secret.fyi"
+  },
+  "record": {
+    "A": [
+      "89.163.215.190",
+      "89.163.215.211"
```

Comment:
both ips don't exist...

Reply:
2nd ip

![image](https://user-images.githubusercontent.com/72417807/114752784-ace4ee80-9d24-11eb-84cc-2a85215b79ff.png)

Reply:
And most likely you're not supposed to access the IP address directly, you're supported to access the website connected to the IP, and then it would show the website and not that error r.

Reply:
Yes, @hackermondev  you are correct!, @TheBotlyNoob its a shared ip, which means its a shared server, Please have some knowledge before reporting